liota/dccs/iotcc.py

```diff
@@ -211,14 +210,22 @@ def _registration(self, msg_id, res_id, res_name, res_kind):
             }
         }
 
-    def _relationship(self, msg_id, parent_res_uuid, child_res_uuid):
+    def _relationship(self, msg_id, parent_entity, child_entity):
         return {
             "transactionID": msg_id,
             "version": self._version,
             "type": "create_relationship_request",
             "body": {
-                "parent": parent_res_uuid,
-                "child": child_res_uuid
+                "parent":  {
+                    "kind": parent_entity.entity_type,
```

Comment:
Please provide comments on why the kind, id and name is required to be sent for _relationship call.

Reply:
As described in the other CL, all the calls have now been standardized to use the same format (uuid, name, kind)

Reply:
Which CL?

Reply:
Corresponding CL for adapter code changes.